lxd/container.go

```diff
@@ -254,22 +254,38 @@ type container interface {
 	NextIdmap() (*idmap.IdmapSet, error)
 }
 
-// Loader functions
+// containerCreateAsEmpty creates an empty instance.
 func containerCreateAsEmpty(d *Daemon, args db.InstanceArgs) (container, error) {
```

Comment:
@stgraber I've noticed that even in the current implementation, if this function returns a non-nil error, although the LXD logs show the operation has failed, the actual lxc cli command `lxc init xxx --empty` doesn't show an error or any indication that it has failed. Could this be a race condition in the operation watching perhaps?

Reply:
@stgraber fixed this here https://github.com/lxc/lxd/pull/6403